### PR TITLE
PR for #58: enable installation from github

### LINKED ISSUE #58: Installing the Github version of the package from inside Stata
@Constantino-Carreto-Romero will look at how we can enable installation from Github inside Stata, as `reghdfe` does. We may need to program a package file. There may also be an alternative using a stata command to install from Github.

### PR DESCRIPTION
**Pull request by Constantino-Carreto-Romero**
Closes #58

### CODE REVIEW

File: dependency.do [hunk 1]
```diff
@@ -0,0 +1,4 @@
+cap ssc install reghdfe
+cap ssc install ftools
+cap ssc install ivreghdfe
+cap ssc install ivreg2
```
Comment (jorpppp) [inline dependency.do:4]: add ranktest here

File: stata.toc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+v 3
+d
+d Simon Freyaldenhoven, Federal Reserve Bank of Philadelphia
+d Christian Hansen, University of Chicago
+d Jorge Pérez Pérez (maintainer), Banco de México (jorgepp@banxico.org.mx)
+d Jesse M. Shapiro, Harvard University
```
Comment (jorpppp) [inline stata.toc:6]: Harvard University and NBER for Jesse's affiliation

**Comment by jorpppp** [comment]
@Constantino-Carreto-Romero will implement some minor changes to the toc and package files.

Once those changes are made, @Constantino-Carreto-Romero will write syntaxes to install the package from the branch or the PR using both `github` and `net` from within Stata. @jorpppp and @rayhuang11 will test this.

**Comment by Constantino-Carreto-Romero** [comment]
I have found a problem that might arise for a user when installing xtevent. This problem arises due to the differences between the files from SSC and the version in the repository. For instance, if the user has installed version 2.0.0 from SSC and then wants to install the version from the repository, the package won't get installed showing the following error:
`
the following files already exist and are different
...
no files installed or copied
`
In the case of installation by `net` this can be solved by adding the `replace` option, but in the case of the `github` command, there is not a `replace` option. Installation can be performed by the `github` command in this case, but there is an extra step the user has to follow.
@jorpppp Should I open an issue for this and add these changes to the README file?

**Pull request merged**